When using the call() and send() functions, 
it doesn't revert if the external call fails.

It is advisable to use transfer() function because it
reverts if the external transaction reverts. 

